internal/cli/atlas/processes/describe.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package processes
+
+import (
+	"context"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const describeTemplate = `ID	REPLICA SET NAME	SHARD NAME	VERSION
+{{.ID}}	{{.ReplicaSetName}}	{{.ShardName}}	{{.Version}}
+`
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	host  string
+	port  int
+	store store.ProcessDescriber
+}
+
+func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.Process(opts.ConfigProjectID(), opts.host, opts.port)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// DescribeBuilder atlas process(es) list --projectId projectId.
+func DescribeBuilder() *cobra.Command {
+	opts := &DescribeOpts{}
+	cmd := &cobra.Command{
+		Use:   "describe <hostname:port>",
```

Comment:
can we pelase add an `Example` to the docs

Reply:
done